raiden-network/light-client
# Adds a changelog reminder github action

**Thank you for submitting this pull request :)**

Fixes #1248 

**Short description**
![image](https://user-images.githubusercontent.com/2269732/80594476-599a6300-8a23-11ea-81e3-742e844351f1.png)



**Definition of Done**


**Steps to manually test the change (dApp)**

1. Create a pull request that modified either `raiden-ts` or `raiden-dapp`, get a comment in the PR that reminds you to add a changelog entry if necessary
2. When changelog entries are added successfully the message should disappear.
